### PR TITLE
Extend message to check for non-minified version

### PR DESCRIPTION
**Pull request by grappler**

### CODE REVIEW

**Comment by jrfnl** [comment]
General question: as this uses a PHPCS "Internal" check, I wonder if we can rely on these type of checks to always be available or if there is a chance that they can be removed/renamed without notice (ok, well a little notice buried somewhere deep in a changelog)

I seem to remember there is another one somewhere in TRTCS as well (or in a branch in progress).

I feel we need to verify whether `Internal` checks can be used in rulesets with the PHPCS project before we start including them.

**Comment by grappler** [comment]
The other one that we have is changes the mixed line endings to error instead of warning.

As PHPCS 3.0 is going to be released soon I don't expect any major changes to PHPCS 2.0 so we should be good. If they were to be removed it would not be the end of the world.

**Comment by jrfnl** [comment]
I would still like to verify the PHPCS stance on usage of these, so that at least we know how their view on this.

**Comment by grappler** [comment]
@jrfnl https://github.com/squizlabs/PHP_CodeSniffer/issues/1256

**Comment by jrfnl** [comment]
@grappler Awesome!

TL;DR: https://github.com/squizlabs/PHP_CodeSniffer/issues/1256 confirms that it is ok to use internal codes in rulesets.

Hereby documented in case of future questions.
